support-frontend/app/views/newspaperArchive.scala.html

```diff
@@ -0,0 +1,23 @@
+@import views.{EmptyDiv, ReactDiv, SSRContent}
+
+@(
+  title: String,
```

Comment:
I suppose passing title from the controller in isn't necessary, we could just hard code it to "Newspaper Archive" in the HTML template. Nota. blocking issue though.